lib/websockex/utils.ex

```diff
@@ -82,6 +84,12 @@ defmodule WebSockex.Utils do
     :sys.handle_debug(debug, &print_event/3, state, event)
   end
 
+  if function_exported?(:sys, :get_log, 1) do
```

Comment:
This fixes the warning due to deprecation of `:sys.get_debug/3` since OTP 22. ([source](https://www.erlang.org/doc/general_info/deprecations.html#functions-deprecated-in-otp-22))